specification/cognitiveservices/data-plane/CustomVision/Training/stable/v3.1/Training.json

```diff
@@ -0,0 +1,5046 @@
+﻿{
+    "swagger": "2.0",
+    "info": {
+        "version": "3.1",
+        "title": "Custom Vision Training Client"
+    },
+    "host": "southcentralus.api.cognitive.microsoft.com",
+    "basePath": "/customvision/v3.1/training",
+    "schemes": [
+        "https"
+    ],
+    "paths": {
+        "/domains": {
+            "get": {
+                "tags": [
+                    "DomainsApi"
+                ],
+                "summary": "Get a list of the available domains.",
+                "operationId": "GetDomains",
```

Comment:
normally operationId is like `Resource_Action`

Reply:
We've not been using that style of naming in cognitive serivces. Renaming the operations isn't something we'd want to do in this change as we're trying not to break existing clients.